app/namespaces/test_adjustment/test_adjustment_service.py

```diff
@@ -233,19 +240,32 @@ def get_adjusted_estimate(self, test_adj, ind_se, ind_sp, ind_se_n, ind_sp_n,
                 upper = None
             else:
                 print('ADJUSTING ESTIMATE', adj_type)
-                model_params = dict(
-                    n_prev_obs=int(denominator_value),
-                    y_prev_obs=int(
-                        serum_pos_prevalence * denominator_value),
-                    n_se=int(output_se_n),
-                    y_se=int(output_se_n * se),
-                    n_sp=int(output_sp_n),
-                    y_sp=int(output_sp_n * sp)
-                )
-                lower, adj_prev, upper = self.pystan_adjust(model_params)
-                if pd.isna(adj_prev):
-                    print(f'FAILED TO ADJUST ESTIMATE')
-
+                # Check bounds of se, se_n, sp, sp_n
+                if se and not (se <= 1 and se >= 0.005):
+                    logging.error("se is not between 0.005 and 1")
+                    lower, adj_prev, upper = None, None, None
+                elif sp and not (sp <= 1 and sp >= 0.005):
+                    logging.error("sp is not between 0.005 and 1")
+                    lower, adj_prev, upper = None, None, None
+                elif output_se_n and output_se_n <= 1:
```

Comment:
These don't need to be bounded between 1 and 1000 - just > 1 is fine

Reply:
what?? it is not bounded between 1 and 1000. im just making sure that output_se_n is not null and then checking if it's greater than 1